gnpy/core/network.py

```diff
@@ -13,6 +13,7 @@
 from gnpy.core import elements
 from gnpy.core.elements import Fiber, Edfa, Transceiver, Roadm
 from gnpy.core.units import UNITS
+from gnpy.core.equipment import *
```

Comment:
We should avoid `import *` in all contexts except possibly in `__init__.py`.

The use of `import *` is controversial in `__init__.py`, and we may also try to avoid it there.

If importing the individual items, as in `from gnpy.core.elements import ...` (above) is too tedious, you can `import gnpy.core.equipment as equipment` and then `equipment.xyz`

Reply:
done, 
it was a remain of my coding investigations